doc/api/repl.markdown

```diff
@@ -237,3 +237,26 @@ The following key combinations in the REPL have these special effects:
   - `<ctrl>D` - Similar to the `.exit` keyword.
   - `<tab>` - Show both global and local(scope) variables
 
+
+### Customizing Object displays in the REPL
+
+The REPL module internally uses
+[`util.inspect`](https://iojs.org/api/util.html#util_util_inspect_object_options),
```

Comment:
The link should be relative. You can see how it's done in https://raw.githubusercontent.com/nodejs/io.js/master/doc/api/console.markdown